edflow/util/edexplore.py

```diff
@@ -53,6 +53,12 @@ def issegmentation(obj):
     bool
         Returns True for rank-three numpy arrays with values boolean or in
         [0,1]
+
+    Warnings
```

Comment:
Maybe we could use the `.. warning::` block here to get nice formatting.

Reply:
Actually I followed the [numpy doc guide](https://numpy.org/devdocs/docs/howto_document.html) because I thought we agreed to follow it, no?

Reply:
True, but I find this: https://numpy.org/devdocs/docs/howto_document.html#other-points-to-keep-in-mind
As far as I understand it, underlined headings are for things the documented functions does. Should the function throw a warning, the place to document that warning would be under the `Warnings` header. But to get a warning text displayed for the reader concerning dangers of the use of the function should, as far as I understand it, be displayed with the `.. warning::` block.

Reply:
Hm, good point but actually there are
- `Warns`: An optional section detailing which warnings get raised and under what conditions, formatted similarly to Raises.
- `Warnings`: An optional section with cautions to the user in free text/reST.
For the `.. warning::` block I find the following: "Use these sparingly, as they do not look very good in text terminals and are not often necessary."
But anyway, both are fine for me - which one do you prefer?

Reply:
Given your comment I would keep it as is and approve!